enterprise/api/license.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+type PlanType string
+
+const (
```

Comment:
api/plan.go already defines this. Maybe move the one to here or put it to common/